src/univariate/continuous/chi.jl

```diff
@@ -79,6 +79,13 @@ function mode(d::Chi; check_args::Bool=true)
     sqrt(ν - 1)
 end
 
+function kldivergence(p::Chi, q::Chi)
+    pv = params(p)
+    qv = params(q)
+    pv2 = pv / 2
```

Comment:
Sorry, my bad, somehow my phone didn't show clearly that you used greek letters here:
```suggestion
    pν = params(p)
    qν = params(q)
    pν2 = pν / 2
```